package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "bfx-facs-hypercore-logger",
+  "version": "1.0.0",
+  "private": false,
+  "description": "Bitfinex Hypercore Logger",
+  "author": {
+    "name": "Michal Kralik",
+    "email": "michal.kralik@bitfinex.com"
+  },
+  "keywords": [
+    "bitfinex",
+    "hypercore",
+    "logger"
+  ],
+  "scripts": {
+    "test": "mocha"
+  },
+  "dependencies": {
+    "bfx-facs-base": "git+https://github.com/bitfinexcom/bfx-facs-base.git",
+    "hypercore-logs": "git+https://github.com/bitfinexcom/hypercore-logs.git",
```

Comment:
please use specific version with tag, e.g. npm i https://github.com/bitfinexcom/hypercore-logs.git#v2.1.0 since the package could have often breaking changes in future